inference-engine/tests_deprecated/functional/vpu/vpu_base/vpu_layer_tests_utils.cpp

```diff
@@ -3,7 +3,7 @@
 //
 
 #include "vpu_layer_tests_utils.hpp"
-
+#include <fstream>
```

Comment:
Is it necessary?

Reply:
yes

Reply:
Could you please explain a bit why? I don't see where you added code to this file that uses something from this header. Maybe I missed something.

Reply:
dependencies between header files. Some includes with fstreams are removed (maybe not in this file, but in includes of this file), so we have to add them explicitly.

Reply:
Thanks for clarification!